data/core/commands/doc.lua

```diff
@@ -67,6 +67,10 @@ end
 
 local function save(filename)
   doc():save(filename)
+  if doc().filename == (USERDIR .. "/init.lua") then
```

Comment:
Probably will not work on Window: / should be replaced by PATHSEP

Reply:
didn't know about PATHSEP, good to know. Will fix it right away.